knexfile.js

```diff
@@ -0,0 +1,54 @@
+// Update with your config settings.

```

Comment:
This file should be in the `server/db` folder. When you pull from `development` you'll get the one that's already there.